libs/api/prisma/migrations/20241022171553_add_migrate_to_target_pm_to_member_plan/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- AlterTable
+ALTER TABLE "member.plans" ADD COLUMN     "migrateToTargetPMid" TEXT;
```

Comment:
Is TEXT here correct shouln't this be VARCHAR? ^^

Reply:
That's how prisma converts it. This is consistent to the rest of the schema. See schema.prisma